src/Bramus/Router/Router.php

```diff
@@ -42,10 +47,56 @@ class Router
      */
     private $serverBasePath;
 
-    /**
-     * @var string Default Controllers Namespace
-     */
-    private $namespace = '';
+	/**
```

Comment:
Why the padding of these lines changed? Did you reformatted/reindented the lines?

Reply:
- [ ] revert improper indention